src/components/AgmExtension/AgmExtensionEvaluation.vue

```diff
@@ -109,18 +109,11 @@ import { DateUtilities } from '@/services'
 export default class AgmExtensionEvaluation extends Vue {
   @Prop({ required: true }) readonly data!: AgmExtEvalIF
 
-  get dueDateString (): string {
-    const date = (DateUtilities.yyyyMmDdToDate(this.data.agmDueDate))
```

Comment:
It's weird that this code isn't reactive, because I think the original example I created recognized when an object project had changed (eg, this.data.agmDueDate).

Anyway, this is OK for now.

Reply:
Gave me a big headache trying to figure out why this was not reactive 👿 Perhaps we can improve this in another ticket.

Reply:
For sure, leave it for now.

It's weird that the new code works, because obviously it's reactive in the template (and then you call a function on it). 🤷‍♂️